docs/Solidity.g4

```diff
@@ -394,7 +394,7 @@ HexDigits
   : HexCharacter ( '_'? HexCharacter )* ;
 
 NumberUnit
-  : 'wei' | 'szabo' | 'finney' | 'ether'
+  : 'wei' | 'gwei' | 'szabo' | 'finney' | 'ether'
```

Comment:
Also should add `gwei` to `identifier` above to remain compatible.

Reply:
Why does gwei need a special treatment here compared to `szabo` or `finney`?

Reply:
Those were tokens from "the beginning of time" or at least from the point on we care about supporting with this grammar.